CenterForOpenScience/osf.io
# [feature] Replace nlogs with brief description text [OSF-8866]

## Purpose
Nlogs aren't useful. Replace them with something more awesomer.


## Changes
1. Remove nlogs and nlogs annotation (hey that's probably a [minor] speed increase too).
2. Add description text that truncates at 150 characters (to be adjusted in demo)
BEFORE:
<img width="588" alt="screen shot 2017-11-13 at 2 07 57 pm" src="https://user-images.githubusercontent.com/1322421/32743992-28483dd8-c87c-11e7-9390-7d5c846e8407.png">
AFTER (No description):
<img width="581" alt="screen shot 2017-11-13 at 2 01 07 pm" src="https://user-images.githubusercontent.com/1322421/32744010-340eee82-c87c-11e7-89ba-2341a2896759.png">

AFTER (<150 chars):
<img width="570" alt="screen shot 2017-11-13 at 2 01 55 pm" src="https://user-images.githubusercontent.com/1322421/32744031-415ecc92-c87c-11e7-8951-7b43417c260e.png">

AFTER (>150 chars):
<img width="567" alt="screen shot 2017-11-13 at 2 00 15 pm" src="https://user-images.githubusercontent.com/1322421/32744045-4a6f63e6-c87c-11e7-9d58-bb0ebe88465b.png">



## Side effects
The log count was hidden for anonymous views. I don't know why. I made this visible.



## Ticket

https://openscience.atlassian.net/browse/OSF-8866

## QA Notes
1. Check empty descriptions
2. Check > X characters (where X is the final cutoff value)
3. Check < X characters

Confirm that the description should be shown for anonymous VOLS. I can't see any reason it shouldn't be shown.
